docs/docs/coverage/os/index.md

```diff
@@ -14,7 +14,7 @@ Trivy supports operating systems for
 | [Alpine Linux](alpine.md)             | 2.2 - 2.7, 3.0 - 3.21, edge         | apk              |
 | [Wolfi Linux](wolfi.md)               | (n/a)                               | apk              |
 | [Chainguard](chainguard.md)           | (n/a)                               | apk              |
-| [Red Hat Enterprise Linux](rhel.md)   | 6, 7, 8                             | dnf/yum/rpm      |
+| [Red Hat Enterprise Linux](rhel.md)   | 6, 7, 8, 9, 10 (partial)            | dnf/yum/rpm      |
```

Comment:
I think listing RHEL 10 (partial) as a supported version might confuse users since it's unclear what is supported. It might be better not to include it just yet, or alternatively, we could add a separate line clearly stating that only SBOM is supported.

```suggestion
| [Red Hat Enterprise Linux](rhel.md)   | 6, 7, 8, 9            | dnf/yum/rpm      |
| [Red Hat Enterprise Linux](rhel.md)   | 10 (SBOM only)            | dnf/yum/rpm      |
```